src/commands/wiki/pizza.ts

```diff
@@ -1,6 +1,8 @@
-import { ChatInputCommandInteraction, SlashCommandBuilder } from "discord.js";
+import { ChatInputCommandInteraction, SlashCommandBuilder, hyperlink } from "discord.js";
 
-import { wikiClient } from "../../kol";
+import { createEmbed } from "../../clients/discord";
+import { pizzaTree } from "../../clients/pizza";
```

Comment:
i want a pizza tree. i am very hungry. (i have not eaten anything today because i have been working since 8am :garf: )